src/Symfony/Component/Runtime/Internal/autoload_runtime.template

```diff
@@ -0,0 +1,18 @@
+<?php
+
+// autoload.php @generated by Symfony Runtime
```

Comment:
Shouldn't this be the resulting filename?
```suggestion
// autoload_runtime.php @generated by Symfony Runtime
```

Reply:
good catch, thanks